lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -103,6 +104,8 @@ class TwilioConnectionTransport extends StateMachine {
     }
 
     const EventPublisher = options.insights ? options.InsightsPublisher : options.NullInsightsPublisher;
+    const iceServerSource = options.overrideIceServers ? { status: 'override', iceServers: options.overrideIceServers } : null;
```

Comment:
@makarandp0 I think this can be simplified:
```js
const iceServers = Array.isArray(options.iceServers) ? options.iceServers : null;
const iceServersStatus = iceServers ? 'override' : 'acquire';

Object.defineProperties(this, {
  _iceServers: {
    value: iceServers,
    writable: true
  }
  _iceServersStatus: {
    value: iceServersStatus
  }
});
```